src/main/kotlin/no/nav/syfo/behandler/BehandlerDialogmeldingService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.syfo.behandler
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.behandler.database.*
+import no.nav.syfo.behandler.database.domain.toBehandlerDialogmeldingBestilling
+import no.nav.syfo.behandler.domain.BehandlerDialogmeldingBestilling
+import java.util.UUID
+
+class BehandlerDialogmeldingService(
+    private val database: DatabaseInterface,
+) {
+    fun getDialogmeldingBestillingListe(): List<BehandlerDialogmeldingBestilling> {
```

Comment:
Her kommer det noe logikk på maks antall sendt-forsøk på sikt?

Reply:
Jeg tror kanskje vi kan se an behovet litt. Det å legge en melding på en MQ-kø vil veldig sjelden feile, men det kan jo være at det å innhente all info'en som trengs for å få en komplett melding er noe som kan skape feilsituasjoner. I så fall er det jo antakelig lurt å operere med et maks antall forsøk ja.

Reply:
Overvåkning av antall feilede forsøk er god kandidat for en alertator-alert som kan gi oss en alert når det går over et visst antall feilede forsøk.